src/Clerk.ts

```diff
@@ -187,18 +187,15 @@ export default class Clerk {
     return decoded as JwtPayload;
   }
 
-  async verifyToken(
-    token: string,
-    algorithms: string[] = ['RS256']
-  ): Promise<JwtPayload> {
+  async verifyToken(token: string): Promise<JwtPayload> {
     const decoded = jwt.decode(token, { complete: true });
     if (!decoded) {
       throw new Error(`Failed to verify token: ${token}`);
     }
 
     const key = await this._jwksClient.getSigningKey(decoded.header.kid);
     const verified = jwt.verify(token, key.getPublicKey(), {
-      algorithms: algorithms as jwt.Algorithm[],
+      algorithms: ['RS256'],
```

Comment:
❓ Are we planning to use this method for other tokens except session token? If yes I think we should keep the option. cc: @chanioxaris, @igneel64